app/src/app/list/polls-table/polls-table.component.sass

```diff
@@ -3,28 +3,45 @@
 @import '../../../sizes'
 
 table
+  display: grid
+  grid-template-columns: 1fr min-content min-content min-content min-content
   border: solid 1px $maincolor
   border-collapse: collapse
+
+
+
   thead
+    display: contents
     border: solid 1px $maincolor
-  td, th
+  tbody,tr
```

Comment:
It would be better to have thead here too. Also please add a space after the commas.